iset.mm

```diff
@@ -64371,6 +64371,12 @@ all reals (greatest real number) for which all positive reals are greater.
       GYQVEUUDYJYSFYGYQYIVHVFUUDYMUUAGYGYQYLVHVFVIYBYCYOEYDYE $.
   $}
 
+  $( Dominance law for disjoint union.  (Contributed by Jim Kingdon,
+     1-Jul-2022.) $)
+  djudom $p |- ( ( A ~<_ B /\ C ~<_ D )
+      -> ( A |_| C ) ~<_ ( B |_| D ) ) $=
+    (  ) ? $.
+
```

Comment:
I'm getting close to the crucial lemma for this (injectivity of the unique H : A |_| B -> X when both F : A -> X and G : B -> X are injective and their ranges are disjoint)...

Reply:
Ooh, thanks for the status report.

After a (less than a day) break that I took for myself, I'm back on the case of what to do about restricted `inl` from the https://github.com/metamath/set.mm/pull/2669#discussion_r916286944 thread. Hopefully this isn't hugely hairy to figure out, but the exmidfodomr proof does rely on the theorems being discussed over there.